### DISCUSSION TITLE
Using mouse wheel to switch virtual desktops

### DISCUSSION THREAD

**Discussion by ikajolroy** [ramensoftware/windhawk#319] [Ideas]
### But this is bad practice for Mac/Linux users. Because Linux or Mac multiple desktop switching is very easy, just ctrl+mouse wheel up down to multiple desktop switching very easy working experience

**Comment by m417z** [comment]
That's not exactly what you're describing, but there's the [Taskbar Scroll Actions](https://windhawk.net/mods/taskbar-scroll-actions) mod that allows switching between virtual desktops by scrolling over the taskbar.

**Reply by ikajolroy** [reply]
yes oh this is already avilable [Taskbar Scroll Actions](https://windhawk.net/mods/taskbar-scroll-actions)
